packages/avatars/README.md

```diff
@@ -9,7 +9,7 @@ This package includes components relating to avatars in the
 npm install @zendeskgarden/react-avatars
 
 # Peer Dependencies - Also Required
-npm install react react-dom prop-types styled-components @zendeskgarden/react-theming
```

Comment:
I believe this should still be a peer-dependency since we are including `propTypes` along with the type defintiions.